### DISCUSSION TITLE
development/16e5vp2y/

### DISCUSSION THREAD

**Discussion by giscus** [cookieY/Yearning#1125] [FAQ]
# development/16e5vp2y/

How to Use Deploy Juno using Docker: In the conf.toml, enter the Juno address in the GrpcAddr parameter (default is 127.0.0.1:50001). Proceed with debugging. Tips Juno shares th...

https://next.yearning.io/development/16e5vp2y/

**Comment by cqupthjb** [comment]
看文档Juno支持linux安装，作者大佬能发Juno下载地址吗？或者将Juno上传到源代码里，多谢~
